extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduOperations.java

```diff
@@ -109,33 +107,36 @@ public boolean deleteAll(
       final String typeName,
       final Short adapterId,
       final String... additionalAuthorizations) {
-    KuduSession session = getSession();
-    try {
-      KuduTable table = getTable(indexName);
-      Schema schema = table.getSchema();
-      List<KuduPredicate> preds =
-          Collections.singletonList(
-              KuduPredicate.newComparisonPredicate(
-                  schema.getColumn(KuduField.GW_ADAPTER_ID_KEY.getFieldName()),
-                  KuduPredicate.ComparisonOp.EQUAL,
-                  adapterId));
-      for (Delete delete : getDeletions(table, preds, KuduRow::new)) {
-        OperationResponse resp = session.apply(delete);
-        if (resp.hasRowError()) {
-          LOGGER.error("Encountered error while deleting all: {}", resp.getRowError());
-        }
-      }
-      return true;
-    } catch (KuduException e) {
-      LOGGER.error("Encountered error while deleting all", e);
-      return false;
-    } finally {
-      try {
-        session.close();
-      } catch (KuduException e) {
-        LOGGER.error("Encountered error while closing Kudu session", e);
-      }
-    }
+    // TODO: this deletion does not currently take into account the typeName, and authorizations are
+    // not supported
+    // KuduSession session = getSession();
```

Comment:
Messed up formatting?

Reply:
Yeah, the autoformatter does not ignore lines commented at column 0. I tried modifying the eclipse-formatter.xml config file to fix this, but it didn't work. Apparently there's a bug in the revelc formatter that causes this to not work, but it should be fixed in the version we're using.